### PR TITLE
Add dracula theme

### PR DESCRIPTION
**Pull request by mikebarkmin**

### CODE REVIEW

File: themes/dracula/dracula.css [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+  color: #D8D8D8 !important;
+  background: #FF79C6 !important;
+  border: none !important;
+  transition: background-color .2s ease-in-out; }
+
+.sidebar-btn-compose:hover {
+  color: #D8D8D8 !important;
+  background: #FF79C6 !important;
+  border: none !important;
+  box-shadow: inset 0 400px 400px rgba(0, 0, 0, 0.1); }
```
Comment (csalmeida) [inline themes/dracula/dracula.css:62]: I have noticed that the contrast ration is not the best for accessibility. Could you change this to another value please? An [`_override.scss` can be created to add theme specific styles](https://github.com/csalmeida/protonmail-themes/tree/master/templates/deutera_one) that do not follow the `_styles.scss` template.

The site recommends `rgb(216, 216, 216)` or `rgb(61, 61, 61)` if this helps but it is absolutely up to you!

Screenshot 2019-10-10 at 11 04 06

> Color contrast test at https://accessible-colors.com

**Review by csalmeida — COMMENTED** [review]
This is awesome @mikebarkmin, thanks for submitting this theme! Just a small change and I think we can merge. 🙏

File: themes/dracula/dracula.css (hunk 1 above)
Comment (mikebarkmin) [reply themes/dracula/dracula.css:62]: Thanks for your suggestion. I have also checked other places where accessibility could be an issue and changes the color accordingly. I hope that everything is fine now.

Comment (csalmeida) [reply themes/dracula/dracula.css:62]: Awesome! LGTM 🚀

**Review by csalmeida — APPROVED** [review]

**Pull request merged**